backend/src/dao/CandidateDeciderDao.ts

```diff
@@ -2,8 +2,70 @@ import { v4 as uuidv4 } from 'uuid';
 import { candidateDeciderCollection, memberCollection } from '../firebase';
 import { DBCandidateDeciderInstance } from '../types/DataTypes';
 import { getMemberFromDocumentReference } from '../utils/memberUtil';
+import BaseDao from './BaseDao';
+
+async function serializeCandidateDeciderInstance(
+  instance: CandidateDeciderInstance
+): Promise<DBCandidateDeciderInstance> {
+  return {
+    ...instance,
+    authorizedMembers: instance.authorizedMembers.map((member) =>
+      memberCollection.doc(member.email)
+    ),
+    candidates: instance.candidates.map((candidate) => ({
+      ...candidate,
+      ratings: candidate.ratings.map((rating) => ({
+        ...rating,
+        reviewer: memberCollection.doc(rating.reviewer.email)
+      })),
+      comments: candidate.comments.map((comment) => ({
+        ...comment,
+        reviewer: memberCollection.doc(comment.reviewer.email)
+      }))
+    }))
+  };
+}
+
+async function materializeCandidateDeciderInstance(
```

Comment:
Let's agree on a naming convention here. Here we call it `materialize....` and `serialize....`, while in DevPortfolio we called this `DBDevPortfolioToDevPortfolio` and `DevPortfolioToDBDevPortfolio`.